src/renderer/modals/ExportOperations/index.js

```diff
@@ -137,7 +139,7 @@ class ExportOperations extends PureComponent<Props, State> {
             ) : (
               <Box>
                 <IconWrapperCircle>
-                  <IconDownloadCloud />
+                  <IconDownloadCloud size={42} />
```

Comment:
#### Icon size
30px seems to be the correct ratio box/svg

```suggestion
                  <IconDownloadCloud size={30} />
```

#### Export operation
Error thrown when canceling the operation
```
Error occurred in handler for 'export-operations': Error: canceled
    at eval (webpack-internal:///./src/main/setup.js:44:11)
    at electron/js2c/browser_init.js:5822:52
    at WebContents.<anonymous> (electron/js2c/browser_init.js:3851:42)
    at WebContents.emit (events.js:210:5)
```

---
2b7244d7 (HEAD -> fix/export-operations-csv-fixes, val/fix/export-operations-csv-fixes) do not show success if you cancel the save file dialog